src/content/docs/cheerpj2/03-getting-started/01-Java-applet.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Run a Java applet
+---
+
+import LinkButton from "../../../../components/LinkButton.astro";
+
+CheerpJ can run Java applets in the browser seamlessly. This page will help you getting started with CheerpJ for Java applets.
+
+**There are two different ways to run a Java Applet in the browser:**
+
+- Running your own Java applet using the CheerpJ runtime environment and the `<cheerpj-applet>` tag in your own webpage.
+- Running a public applet using the [CheerpJ Applet Runner](https://chrome.google.com/webstore/detail/cheerpj-applet-runner-bet/) Chrome extension for applets integrated with the applet tag `<applet>` on public websites.
```

Comment:
Link to each section here, or maybe make the public applet section a different page altogether

Reply:
I'll add the links for now. I am considering to separate this with tabs or so.